app/models/post.rb

```diff
@@ -0,0 +1,12 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+
+  def update_posts_counter
+    author.update(posts_counter: author.posts.count)
+  end
```

Comment:
- **[OPTIONAL]** We can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/220232750-7fb9f2f9-55b9-4b9e-bda4-837ff55371d1.png)